lib/utils/exception_classifier.dart

```diff
@@ -0,0 +1,9 @@
+String classifyException(String exceptionString) {
+  if (exceptionString.contains('ClientException')) {
```

Comment:
Is ClientException related to client to connect to the API?

Reply:
Yes @dellyjm, the ClientException is thrown when a user tries to connect the API.